handbook/product/tracking_issues.md

```diff
@@ -0,0 +1,88 @@
+
+## Planning a milestone with a tracking issue
+
+Planning happens in the last week of the previous milestone.
+
+1. [Open a new tracking issue](#creating-a-tracking-issue) with the appropriate `team/$TEAM` label and the right milestone.
+1. Ask all members of the team through appropriate channels to create or assign issues to this milestone. When possible, issues should be estimated with the appropriate `estimate/$DAYS` label.
+1. Once everyone's ready, have the tracking issue reviewed by relevant parties (e.g. product team, VP of Engineering).
+1. Make adjustments to the planned work with the team based on the feedback from the review.
+1. Add the `planned/$MILESTONE` label to all planned issues. This is used to track what work was originally planned, even if it gets deprioritized during the milestone.
+
+All planned work items are meant to be finished in the iteration. If this doesn't happen, it is either due to unknown unknowns or poor estimating accuracy. While we cannot work on the former, we can improve the latter — over time, we aim to estimate better.
+
+Teammates should be conservative by picking an estimate somewhere in between the median case and worst case scenarios, not best case. The higher someone's level of experience with the issue being estimated as well as estimation itself, the closer the estimate can be to the best case scenario. The opposite is also true.
+
+Despite estimation having a bad reputation in some circles, we find it valuable to:
+
+1. Prevent any single team member having more work assigned than available work days.
+1. Ensure everyone has enough slack in the iteration (~20% of their time). This is to account for unexpected things, to foster creativity and to prevent burn-out.
+1. Remind the person working on the issue to avoid time sinks or approaches that would miss the estimate.
+
+Roadmap items may take more than an iteration to implement. As such please link to an issue which is scoped to this iteration. If it isn't clear how to break down a larger item in a scope increment, a **spike** of one or two days could be appropriate to determine such scope and then create the resulting issues for the iteration — it's fine to update the tracking issue with the results of a spike during the iteration.
+
+## Using a Tracking issue for progress check-ins
+
+A Tracking issue can be used as a central communication hub for the progress being done around whatever is being tracked.
+
+To that end, those working on a tracking issue write high level periodic summaries of their progress for interested parties and other stake-holders to follow along without being exposed to too much detail. These summaries are written as comments on the tracking issue.
```

Comment:
```suggestion
To that end, those working on a tracking issue write high level periodic summaries of their progress for interested parties and other stakeholders to follow along without being exposed to too much detail. These summaries are written as comments on the tracking issue.
```

I don't know, but "stakeholders" is what we use [most of time in this repository](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/about%24+stakeholders&patternType=literal), and ["stake-holders" only has one occurrence](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/about%24+stake-holders&patternType=literal).